client/src/components/AddCard/AddCardForm/useStyles.ts

```diff
@@ -0,0 +1,48 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles((theme) => ({
+  form: {
+    width: '100%', // Fix IE 11 issue.
+    marginTop: theme.spacing(1),
+    alignItems: 'center',
```

Comment:
Question: Aren't these properties only applied to the flex containers, they shouldn't have any effect here?

Reply:
@samsoncsyu7777 please respond to the above comment :)

Reply:
I missed this comment before. Done.